other/README.md

```diff
@@ -48,15 +48,16 @@ Important and useful :link: links to other documentation and standards at NYPL.
 ## Web Applications
 
 ### React/Node
+* Header
+  * [App Repository](https://bitbucket.org/NYPL/nypl-dgx-react-header/)
```

Comment:
The bitbucket repos are deprecated and the Github ones are the correct ones.

Reply:
Oh, good catch! Thanks very much @EdwinGuzman

Reply:
Fixed now